File: src/G1_rust/PYA107.rs
use std::io;

fn main(){
    let mut input: String = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let num1: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let num2: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let num3: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let num4: f64 = input.trim().parse().unwrap();
    input.clear();
    io::stdin().read_line(&mut input).unwrap();
    let num5: f64 = input.trim().parse().unwrap();
    input.clear();

    let sum: f64 = num1+num2+num3+num4+num5;
    let avg: f64 = sum/5.0;

    println!("{} {} {} {} {}",num1,num2,num3,num4,num5);
    println!("Sum = {:.1}",sum);
    println!("Average = {:.1}",avg);

}

File: src/G1_py/PYA107.py

num1 = eval(input())
num2 = eval(input())
num3 = eval(input())
num4 = eval(input())
num5 = eval(input())

sum = num1+num2+num3+num4+num5
avg = sum/5

print("{:} {:} {:} {:} {:}".format(num1,num2,num3,num4,num5))
print("Sum = {:.1f}".format(sum))
print("Average = {:.1f}".format(avg))